src/docs/user/references/ExecutionSchedulerServiceReference.adoc

```diff
@@ -0,0 +1,62 @@
+
+=== Introduction
+The <<_glossary_execution_scheduler,*Execution Scheduler*>> schedules <<_glossary_workflow,*Workflows*>>. Given a
+ A <<_glossary_workflow,*Workflow*>> is scheduled by the <<_glossary_execution_scheduler,*Execution Scheduler*>> if
+  an <<_glossary_execution_calendars_object,*Execution Calendars Object*>> is attached to the
+   <<_glossary_generic_information,*Generic Information*>> of a <<_glossary_workflow,*Workflow*>>.
+
+=== Execution Calendars Object Syntax
+Below is an example <<_glossary_execution_calendars_object,*Execution Calendars Object*>>. It has a cron
+expression attached as "cron".
+----
+EXECUTION_CALENDARS =
+{
+    "cron": "0 23 * * MON-FRI",
+}
+----
+One can see the cron expression "0 23 * * MON-FRI", which can thoroughly explained in the
+<<_execution_scheduler_cron_expression_syntax, Cron Expression Syntax>> section.
+
+=== Execution Scheduler Submission
+When you are using <<_glossary_workflow_studio,*Workflow Studio*>>, add an
+<<_glossary_execution_calendars_object,*Execution Calendars Object*>> to <<_glossary_generic_information,*Generic Information*>>
+of a <<_glossary_workflow,*Workflow*>> and click submit.
+ The <<_glossary_workflow_studio,*Workflow Studio*>> automatically detect the
+ <<_glossary_execution_calendars_object,*Execution Calendars Object*>> and submits the <<_glossary_workflow,*Workflow*>>
+ accoordingly.
+
+If you manually submit <<_glossary_workflow,*Workflow*>>, you need to submit it to the REST API of the
+<<_glossary_execution_scheduler,*Execution Scheduler*>>.
+
+The proactive-client does currently not support <<_glossary_execution_scheduler,*Execution Scheduler*>>
+ <<_glossary_workflow,*Workflows*>>.
+
+
+[[_execution_scheduler_cron_expression_syntax]]
+==== Cron Expression Syntax
+The syntax of the follows the UNIX crontab pattern, a string consists of five space-separated segments.
+ Please refer http://www.sauronsoftware.it/projects/cron4j/manual.php#p02[cron4j documentation] for further details.
+
+
+EXECUTION_CALENDARS =
+{
+    "cron": "0 23 * * MON-FRI",
```

Comment:
Also what do you think about the syntax limitation during the input with &quote; ? do you find any solution, otherwise, i think we should specify it here in the doc